makefile.shade

```diff
@@ -121,7 +121,7 @@ var buildTarget = "compile"
             blockLogger.StartBlock("Cloning repos");
         }
 
-        Parallel.ForEach(repositories, repo =>
+        Parallel.ForEach(repositories, new ParallelOptions { MaxDegreeOfParallelism = threads }, repo =>
```

Comment:
Is `threads` defined in this target? Think you might have to copy the line from `ci-build`